### PR TITLE
Fix default on quickstart test and add bootstrap script for examples and tests

### PR DESCRIPTION
**Pull request by DoriftoShoes**
This adds a script to bootstrap the examples or tests packs that are included in contrib.  This will be used by the e2e deployment workflows.

### CODE REVIEW

File: tools/st2-setup-samples.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+PACK=$1
+if [[ "$PACK" == "examples" ]]; then
+    PATH="/usr/share/doc/st2/"
+elif [[ "$PACK" == "tests" ]]; then
+    PATH="/usr/share/stackstorm/"
+fi
+
+echo -e "Adding packs_base_paths for ${PATH}/${PACK}..."
+grep packs_base_path /etc/st2/st2.conf || echo -e "\n[content]\npacks_base_paths = ${PATH}/${PACK}" >> /etc/st2/st2.conf'
```
Comment (Kami) [inline tools/st2-setup-samples.sh]: I know this is not an issue in this case, but there is a double slash between path and pack.

File: tools/st2-setup-samples.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+PACK=$1
+if [[ "$PACK" == "examples" ]]; then
+    PATH="/usr/share/doc/st2/"
+elif [[ "$PACK" == "tests" ]]; then
+    PATH="/usr/share/stackstorm/"
+fi
```
Comment (Kami) [inline tools/st2-setup-samples.sh]: Would be good to add else and throw / exit if uses passes in an invalid / unsupported pack name.

```bash
...
else:
  echo "Invalid pack: ${PACK}
  exit 2
fi
...
```

File: tools/st2-setup-samples.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+echo -e "Adding packs_base_paths for ${PATH}/${PACK}..."
+grep packs_base_path /etc/st2/st2.conf || echo -e "\n[content]\npacks_base_paths = ${PATH}/${PACK}" >> /etc/st2/st2.conf'
+
+if [[ $? ne 0 ]]; then
```
Comment (Kami) [inline tools/st2-setup-samples.sh]: Missing dash (`-ne`)

File: tools/st2-setup-samples.sh (hunk 1 above)
Comment (Kami) [inline tools/st2-setup-samples.sh]: Extra single quote at the end of the line :P

**Comment by DoriftoShoes** [comment]
It was late...stop making fun of me :P.  I'm fixing this stuff now and adding virtualenv creation in case we add any python actions.

File: tools/st2-setup-samples.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```
Comment (dzimine) [inline tools/st2-setup-samples.sh]: Suggestion: only install tests pack, and leave installing examples up to the tests, as needed.

**Comment by Kami** [comment]
@DoriftoShoes I know, sorry, nothing personal :P

**Comment by Kami** [comment]
LGTM, +1

**Pull request merged**